sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -173,8 +173,8 @@ int hda_dsp_ctrl_clock_power_gating(struct snd_sof_dev *sdev, bool enable)
 					HDA_VS_INTEL_EM2_L1SEN, val);
 
 	/* enable/disable audio dsp power gating */
-	val = enable ? 0 : PCI_PGCTL_ADSPPGD;
-	snd_sof_pci_update_bits(sdev, PCI_PGCTL, PCI_PGCTL_ADSPPGD, val);
+	val = enable ? 0 : (PCI_PGCTL_ADSPPGD | BIT(1));
+	snd_sof_pci_update_bits(sdev, PCI_PGCTL, PCI_PGCTL_ADSPPGD | BIT(1), val);
```

Comment:
should we add a macro for `BIT(1)`?

Reply:
is this change really necessary?

Reply:
It is not needed, leftover from the million rounds of trying to figure out what is going on..